content/docs/authorization/auth-code-flow/index.md

```diff
@@ -1,493 +1,90 @@
 ---
-layout: layout-sidebar
 name: Authorization Code Flow
 description: The Authorization Code Flow is appropriate for applications that the user logs into once. Since this flow involves an exchange of your **Application secret* for an access token, it is suitable for applications than run from secure locations such as server-side web application or back-end service.
+layout: layout-tutorial
 order: 100
 published: true
 showInNav: true
-back_to_top: true
 title: Authorization Code Flow
+sidebarLayoutSecondaryColumn: col-md-2 sidebar-tutorial
+sidebarLayoutPrimaryColumn: col-md-8
+bootstrap_container: bg-tutorial
+markdown: false
```

Comment:
The reason I converted the pages to `markdown: false` was to make sure the page displayed correctly with the pattern and not have extra spaces. I can make sure the includes files we use are in markdown.